_posts/fr/newsletters/2022-09-07-newsletter.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Bitcoin Optech Newsletter #216'
+permalink: /fr/newsletters/2022/09/07/
+name: 2022-09-07-newsletter-fr
+slug: 2022-09-07-newsletter-fr
+type: newsletter
+layout: newsletter
+lang: fr
+---
+Cette semaine au sommaire de cette newsletter, plusieurs changements notables au
+logiciel de l'infrastructure Bitcoin.
+
+## Nouvelles
+
+*Pas de nouvelles signifiantes cette semaine.*
+
+## Changements notables dans le code et la documentation
+
+*Changements notables cette semaine dans [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25717][] Ajout d'une étape de pré-synchronisation des entêtes
+  (Headers Presync) durant le téléchargement initial des blocs (Initial Block
+  Download - IBD) afin de prévenir d'une attaque par déni de service (DoS) ainsi
+  qu'une étape vers la suppression des points de contrôle. Les noeuds utilisent
+  la pré-synchronisation pour vérifier que la chaîne d'en-têtes d'un pair a
+  suffisamment travaillé avant de la stocker de manière permanente.
+
+  Au cours de l’IBD, un pair malicieux pourrait essayer de bloquer le process de
+  synchronisation, injecter des blocs qui ne sont pas de la chaîne la plus
+  longue, ou simplement épuiser les ressources du noeud. Ainsi, bien que la
+  vitesse de synchronisation et l'usage de la bande passante soient des
+  préoccupations importantes pendant l'IBD, un objectif principal de conception
+  est d'éviter l'attaque par déni de service (DoS). Depuis la version v0.10.0,
+  le noeud Bitcoin Core synchronise d'abord la première en-tête de block avant
+  de télécharger les données du bloc et rejette les en-têtes qui ne respectent
+  pas un ensemble de points de contrôle.
```

Comment:
It's great that you are doing this @Copinmalin. That said, I would encourage you to check carefully for inaccuracies and errors a few hours or a day later after getting some distance. It's not clear to me whether these are issues with translation  or misunderstanding of some of the nuance in the source content. A few examples in one paragraph follow. Thanks for the excellent initiative!

```diff
-  Au cours de l’IBD, un pair malicieux pourrait essayer de bloquer le process de
+  Au cours de l’IBD, un pair malicieux pourrait essayer de ralentir le process de
   synchronisation, injecter des blocs qui ne sont pas de la chaîne la plus
   longue, ou simplement épuiser les ressources du noeud. Ainsi, bien que la
   vitesse de synchronisation et l'usage de la bande passante soient des
-  préoccupations importantes pendant l'IBD, un objectif principal de conception
-  est d'éviter l'attaque par déni de service (DoS). Depuis la version v0.10.0,
-  le noeud Bitcoin Core synchronise d'abord la première en-tête de block avant
-  de télécharger les données du bloc et rejette les en-têtes qui ne respectent
-  pas un ensemble de points de contrôle.
+  préoccupations importantes pendant l'IBD, un objectif principal reste
+  l'évitement des attaques par déni de service (DoS). Depuis la version v0.10.0,
+  Bitcoin Core synchronise d'abord les en-têtes des blocks avant
+  de télécharger les données des blocks et rejette les en-têtes qui ne respectent
+  devient des checkpoints de consensus.
```